Dockerfile

```diff
@@ -126,8 +126,8 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     tk-dev \
     && sudo rm -rf /var/lib/apt/lists/*
 
-ARG PYTHON_VERSION=3.10.8
-ARG PYENV_GIT_TAG=v2.3.7
+ARG PYTHON_VERSION=3.11.4
```

Comment:
Please explain in more detail in the commit message why bumping all of these tool versions is required. Is it really that all the previous versions did not support Python 3.11 yet?

Reply:
I cannot explain this for all tools, only for Pyenv and Poetry. Should I downgrade them and test it?

Reply:
Well, basically yes. Or put the other way: You should have only upgraded those tools which are required to be updated in this context / to work with Python 3.11.

Reply:
Update of not relevant tools reverted.